septentrion/runner.py

```diff
@@ -41,7 +41,7 @@ def close(self) -> None:
             raise SQLRunnerException("Block closed !")
         self.closed = True
 
-    def run(self, cursor: psycopg2.extentions.cursor) -> int:
+    def run(self, cursor: cursor) -> int:
```

Comment:
the cursor parameter overrides the cursor object imported. There is no problem here but is it a good practice ?

Reply:
oh gosh you're right ! Not sure what the rest of the world thinks of it but I hate it, it creates ambiguity.

Reply:
maybe we could do `from psycopg2.extensions.cursor import cursor as Cursor` ?

Reply:
yes, that was my planned fix. This or `as cursor_class`, but what you suggested was my number 1.